### PR TITLE
Remove old cruft.

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

File: pdcupdater/handlers/modules.py [hunk 1]
```diff
@@ -38,12 +38,7 @@ class ModuleStateChangeHandler(pdcupdater.handlers.BaseHandler):
     @property
     def topic_suffixes(self):
         return [
-            # This is the modern value that will be used.
-            # See https://pagure.io/fm-orchestrator/pull-request/279
             'mbs.module.state.change',
-            # This is an old value for backwards compat during development.
-            # Feel free to remove it in the future.
```
Comment (bowlofeggs) [inline pdcupdater/handlers/modules.py]: I see just how free you feel!

Comment (ralphbean) [inline pdcupdater/handlers/modules.py]: :eagle:

**Comment by ralphbean** [comment]
Thanks @bowlofreview!

**Comment by bowlofeggs** [comment]
On Tue, 2017-03-14 at 17:06 -0700, Ralph Bean wrote:
> Thanks @bowlofreview!

No @bowlofproblem!

**Pull request merged**